### ISSUE TITLE
Bug with commitMessagePrefix in latest Release

### ISSUE THREAD

**Issue by halbekanne** [renovatebot/renovate#16269]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

32.99.5

### Please select which platform you are using if self-hosting.

Bitbucket Server

### If you're self-hosting Renovate, tell us what version of the platform you run.

7.17

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the bug

I have a commitMessagePrefix defined as "🆙". When using the latest release, the commit message and PR title is not correct.

**Actual:**
🆙: 🆙 Update plugin org.owasp.dependencycheck to v7

**Expected / before the bug:**
🆙 Update plugin org.owasp.dependencycheck to v7

This used to work like expected last week, so the bug was probably introduced in one of the latest releases.
I've tested it with an older version and the expected behavior was still present with version 32.90.0

### Relevant debug logs

<details><summary>Logs</summary>

```
DEBUG: getBranchPr(renovate/org.jboss.weld-weld-junit4-4.x) (repository=<redacted>, branch=renovate/org.jboss.weld-weld-junit4-4.x)
DEBUG: findPr(renovate/org.jboss.weld-weld-junit4-4.x, "undefined", "open") (repository=<redacted>, branch=renovate/org.jboss.weld-weld-junit4-4.x)
DEBUG: getPrList() (repository=<redacted>, branch=renovate/org.jboss.weld-weld-junit4-4.x)
DEBUG: returning cached PR list (repository=<redacted>, branch=renovate/org.jboss.weld-weld-junit4-4.x)
DEBUG: Renovate did not find a PR for branch #renovate/org.jboss.weld-weld-junit4-4.x (repository=<redacted>, branch=renovate/org.jboss.weld-weld-junit4-4.x)
DEBUG: branchExists=false (repository=<redacted>, branch=renovate/org.jboss.weld-weld-junit4-4.x)
DEBUG: dependencyDashboardCheck=undefined (repository=<redacted>, branch=renovate/org.jboss.weld-weld-junit4-4.x)
DEBUG: recreateClosed is false (repository=<redacted>, branch=renovate/org.jboss.weld-weld-junit4-4.x)
DEBUG: findPr(renovate/org.jboss.weld-weld-junit4-4.x, ":  Update dependency org.jboss.weld:weld-junit4 to v4", "!open") (repository=<redacted>, branch=renovate/org.jboss.weld-weld-junit4-4.x)
DEBUG: getPrList() (repository=<redacted>, branch=renovate/org.jboss.weld-weld-junit4-4.x)
DEBUG: returning cached PR list (repository=<redacted>, branch=renovate/org.jboss.weld-weld-junit4-4.x)
DEBUG: Renovate did not find a PR for branch #renovate/org.jboss.weld-weld-junit4-4.x (repository=<redacted>, branch=renovate/org.jboss.weld-weld-junit4-4.x)
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by bobvandevijver** [comment]
I'm seeing exactly the same, due to this all closed MRs for rejected updated were recreated as well 😔

**Comment by wyardley** [comment]
Thanks, same issue:
https://github.com/renovatebot/renovate/discussions/16275

**Comment by dfeldick** [comment]
Same issue. My renovate.json is like this because I need my gitlab branch names and commit messages to match a pre-defined pattern:
{
"branchPrefix": "APIGOV-renovate-",
"commitMessagePrefix": APIGOV-renovate -"
...
}

On Friday, June 24, my branch names looked like this:
APIGOV-renovate-google.golang.org-protobuf-1.x

and a corresponding merge request looked like this:
APIGOV-renovate - Update module google.golang.org-protobuf-1.x

Sometime this weekend my scheduled runs of the bot, and continuing today with manual runs, my branch names look like this:
APIGOV-renovate-google.golang.org-protobuf-1.x

The branch names are still correct. But, my MR titles look like this:
APIGOV-renovate -: aPIGOV-renovate - Update module google.golang.org-protobuf-1.x

This is wrong. Somewhere along the way it appears that the ':' was added, the commitMessagePrefix has been duplicated, and the capitalization has been messed with.

**Comment by ahatzz11** [comment]
I can also confirm that this is happening as of the latest release. With the following configuration:

```
{
  "commitMessagePrefix": "`catalog` - "
}
```

On Friday my PR titles were correct:
> `catalog` - Update plugin org.jetbrains.kotlin.jvm from `v1.6.10` to `v1.7.0`

Today I am seeing messages that are formatted strangely:
> `catalog` -: `catalog` - Update plugin org.jetbrains.kotlin.jvm from `v1.6.10` to `v1.7.0`

**Comment by rarkins** [comment]
Fixed in #16263 but still awaiting a release

**Comment by HonkingGoose** [comment]
This should be fixed in [release `32.99.6`](https://github.com/renovatebot/renovate/releases/tag/32.99.6).
